hazelcast/src/main/java/com/hazelcast/client/impl/clientside/ClientDynamicClusterConfig.java

```diff
@@ -288,6 +288,7 @@ public Config addExecutorConfig(ExecutorConfig executorConfig) {
         return this;
     }
 
+    // TODO stats
```

Comment:
Leftover TODOs?

Reply:
Realised it's about the client protocol. Can you add it regardless?

Reply:
will address in a later PR.